crates/op-rbuilder/src/flashtestations/args.rs

```diff
@@ -23,17 +23,24 @@ pub struct FlashtestationsArgs {
     )]
     pub debug: bool,
 
-    // Debug url for attestations
-    #[arg(long = "flashtestations.debug-url", env = "FLASHTESTATIONS_DEBUG_URL")]
-    pub debug_url: Option<String>,
+    // Debug static key for the tee key
+    #[arg(
+        long = "flashtestations.debug-tee-key-seed",
+        env = "FLASHTESTATIONS_DEBUG_TEE_KEY_SEED",
+        default_value = "debug"
+    )]
+    pub debug_tee_key_seed: String,
```

Comment:
Should this be Option<String>?

Reply:
it has a default value and won't be used if debug not enabled so it won't be a required arg. Option<String> will be same logic by having .unwrap_else("debug")